examples/ImageRecon/GEOmetrics/architectures.py

```diff
@@ -244,32 +233,27 @@ def reset_parameters(self):
             self.bias.data.uniform_(-.1, .1)
 
     def forward(self, input, adj):
-        
         support = torch.mm(input, self.weight1)
-        side_len = max(support.shape[1]//3, 2)
+        side_len = max(support.shape[1] // 3, 2)
         if adj.type() == 'torch.cuda.sparse.FloatTensor': 
-         
-            norm = torch.sparse.mm(adj,torch.ones((support.shape[0], 1)).cuda())
-            normalized_support = support[:, :side_len] /norm
+
+            norm = torch.sparse.mm(adj, torch.ones((support.shape[0], 1)).cuda())
+            normalized_support = support[:, :side_len] / norm
             side_1 = torch.sparse.mm(adj, normalized_support)
         else: 
             side_1 = torch.mm(adj, support[:, :side_len])
-        
-        side_2 = support[:,side_len: ]
-        output = torch.cat((side_1, side_2), dim = 1)
-        
+
+        side_2 = support[:, side_len:]
+        output = torch.cat((side_1, side_2), dim=1)
+
         if self.bias is not None:
             output = output + self.bias
         return output
 
     def __repr__(self):
         return self.__class__.__name__ + ' (' \
```

Comment:
Not very important but this can fit into one line

Reply:
While you're correct, I'll leave this as is - defining side_2 makes it a clearer what is being concatenated.